examples/transformerconfigs/README.md

```diff
@@ -44,26 +51,30 @@ commonLabels:
 - path: spec/selector/matchLabels
   create: true
   kind: Deployment
-  (etc.)  
+# add additional paths to resource fields
 ```
 
-## name reference transformer
-Name reference transformer's configuration is different from all other transformers. It contains a list of namebackreferences, which represented all the possible fields that a type could be used as a reference in other types of resources. A namebackreference contains a type such as ConfigMap as well as a list of FieldSpecs where ConfigMap is referenced. Here is an example.
-```
+## Name reference transformer
+
+Name reference transformer's configuration is different from all other transformers. It contains a list of `nameReferences`, which represent all of the possible fields that a type could be used as a reference in other types of resources. A `nameReference` contains a type such as ConfigMap as well as a list of `fieldSpecs` where ConfigMap is referenced in other resources. Here is an example.
+
+```yaml
 kind: ConfigMap
 version: v1
-FieldSpecs:
+fieldSpecs:
 - kind: Pod
   version: v1
   path: spec/volumes/configMap/name
 - kind: Deployment
   path: spec/template/spec/volumes/configMap/name
 - kind: Job
   path: spec/template/spec/volumes/configMap/name
-  (etc.)
-```
-Name reference transformer configuration contains a list of such namebackreferences for ConfigMap, Secret, Service, Role, ServiceAccount and so on.
+# add additional paths to resource fields
```

Comment:
Same as above. How about `truncated` or not adding the comment at all?

Reply:
I guess no comment will do.